website/docs/r/slo.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: 'sumologic' page_title: 'SumoLogic: sumologic_slo' description: |- Provides the ability to create, read, delete,
+and update SLOs.
+---
+
+# sumologic_slo
+
+Provides the ability to create, read, delete, and update SLOs.
+
+## Example SLO
+
+```hcl
+resource "sumologic_slo" "slo_tf_test" {
+  name        = "slo-tf-test1"
+  description = "example SLO created with terraform"
+  parent_id   = "0000000000000001"
+  signal_type = "Latency"
+  service     = "auth"
+  application = "login"
+  compliance {
+    compliance_type = "Rolling"
+    size            = "7d"
+    target          = 99
+    timezone        = "Asia/Kolkata"
+  }
+  indicator {
+    evaluation_type = "Window"
+    op              = "LessThan"
+    query_type      = "Metrics"
+    size            = "1m"
+    threshold       = 200
+    queries {
+      query_group_type = "Threshold"
+      query_group {
+        row_id = "A"
+        query  = "metric=request_time_p90  service=auth api=login"
+      }
+    }
+
+  }
+}
+```
+
+## SLO Folders
+
+NOTE: Monitor folders are considered a different resource from Library content folders. See [sumologic_slo_folder][2]
+for more details.
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the SLO. The name must be alphanumeric.
+- `description` - (Optional) The description of the SLO.
+- `parent_id` - (Optional) The ID of the SLO Folder that contains this SLO. Defaults to the root folder.
+- `signal_type` - (Optional) The type of SLO. Valid values are `Latency`, `Error`,`Throughput`,`Availability`
+  , `Other`. Defaults to `Latency`.
+- `service` - (Optional) The notifications the SLO will send when the respective trigger condition is met.
+- `application` - (Optional) Whether to group notifications for individual items that meet the trigger condition.
+  Defaults to true.
+- `compliance` - (Required) The compliance settings for the SLO.
+    - `compliance_type` - (Required) The type of compliance to use. Valid values are `Rolling` or `RollingWithWindow`.
+    - `target` - (Required) The target value to use, must be a number between 0 and 100.
+    - `timezone` - (Required) Time zone for the SLO compliance. Follow the format in the [IANA Time Zone Database][3].
+    - `size` - (Required) The size of the window to use, minimum of `1d` and maximum of `14d`.
+- `indicator` - (Required) The indicator that defines the conditions of when to send notifications.
+    - `evaluation_type` - (Required) Evaluate SLI using successful/total windows, or occurrence of successful events
+      over entire compliance period.. Valid values are `Window` or `Request`.
+    - `op` - (Required) The operator to use. Valid values are `LessThan`, `LessThanOrEqual`, `GreaterThan`
```

Comment:
Can be improved a bit, maybe `The operator used to define a successful window / event`